libheif/CMakeLists.txt

```diff
@@ -89,3 +89,20 @@ if(X265_FOUND)
     target_link_libraries(${PROJECT_NAME} ${X265_LIBRARIES} ${X265_LINKDIR})
 endif()
 
+# Install commands
+# --------------------------
+write_basic_package_version_file(heifConfigVersion.cmake COMPATIBILITY ExactVersion)
+
+install(TARGETS ${PROJECT_NAME} EXPORT heifConfig
```

Comment:
Please use `${PROJECT_NAME}-config`

Reply:
Right now I am using `${PROJECT_NAME}Config` which also works. Note that the CMake documentation mentions that both format are accepted:

https://cmake.org/cmake/help/v3.16/command/find_package.html#full-signature-and-config-mode

Is there some limitation about the current name you are aware of?